app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -36,14 +37,30 @@ def attach_question_flow(questions)
     end
   end
 
+  def upload_drawings(drawing_params)
+    unless drawing_params.nil?
+    begin
+    drawing_params.each do |param|
+      drawing = @planning_application.drawings.create(tags: Array(param[:tags]))
+      drawing.plan.attach(io: File.open(open(param[:filename])), filename: "#{new_plan_filename(param[:filename])}")
+    rescue
+    end
+    end
+    end
```

Comment:
This is interesting indentation. Surprised this is how rubocop wants it

Reply:
Running Rubocop without -a meant that it still passed AND looks better :) But yes, really odd that it autocorrected to this!